pandas/core/reshape/encoding.py

```diff
@@ -316,3 +318,202 @@ def get_empty_frame(data) -> DataFrame:
             dummy_mat = dummy_mat[:, 1:]
             dummy_cols = dummy_cols[1:]
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
+
+
+def from_dummies(
+    data: DataFrame,
+    sep: None | str = None,
+    base_category: None | Hashable | dict[str, Hashable] = None,
```

Comment:
can we call this default_category? (maybe more obvious), not sure how we got to this name

Reply:
The name was [changed to base_category after these thougts](https://github.com/pandas-dev/pandas/pull/41902#discussion_r796182012) as I got the name from the [wiki article on dummy variables](https://en.wikipedia.org/wiki/Dummy_variable_(statistics)#Incorporating_a_dummy_independent). I thought this might be a place where people, who are not already familiar with the concept, are likely to stop for a read.

Reply:
I would be slightly more partial to `default_category`